cpp/tests/interop/from_arrow_test.cpp

```diff
@@ -89,6 +89,30 @@ TEST_F(FromArrowTest, DateTimeTable)
   cudf::test::expect_tables_equal(expected_table_view, got_cudf_table->view());
 }
 
+TEST_F(FromArrowTest, DurationTable)
```

Comment:
consider parameterizing the test to cover all supported time units.

Reply:
Parametrized the test 👍